s3tests_boto3/functional/test_s3.py

```diff
@@ -2817,6 +2817,54 @@ def test_post_object_upload_size_below_minimum():
 
     r = requests.post(url, files=payload, verify=get_config_ssl_verify())
     assert r.status_code == 400
+    assert r.error_code == 'EntityTooSmall'
```

Comment:
this fails with `AttributeError: 'Response' object has no attribute 'error_code'`

this is a call to `requests.post()` which doesn't understand these s3 error codes. i think we'd have to parse the xml response to get at it. i'm going to remove this commit for now